app/views/works/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="new_page">
+  <%= form_for @work do |f| %>
+  <ul class="new_info">
+    <li>
+      <%= f.label :categoty %>
+      <%= f.select "category", raw("<option>Album</option><option>Book</option><option>Movie</option>") %>
+    </li>
```

Comment:
This dropdown will send capitalized media categories, while your model expects lowercase. That's probably why creation failed.